### DISCUSSION TITLE
Archery-1.10.0 release docker-compose文件镜像tag号

### DISCUSSION THREAD

**Discussion by hbzhu** [hhyo/Archery#2301] [General]
Release v1.10.0两个压缩包，查看了Archery-1.10.0/src/docker-compose/docker-compose.yml的内容为：

archery:
    image: hhyo/archery:v1.9.1

应该是
archery:
    image: hhyo/archery:v1.10.0

**Comment by LeoQuote** [comment]
docker-compose 仅为最小实践, 版本需要根据你自己需要来, 欢迎pr
